pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -43,6 +43,15 @@
 warnings.filterwarnings("ignore")
 LOGGER = get_logger()
 
+DATETIME_INDEX_MODELS = [
```

Comment:
Hi @TremaMiguel Will you revert back this change. It should not be needed with if you are converting the datetime_index to period_index to all models in setup and handling Prophet using the wrapper.